src/main/java/uk/gov/hmcts/reform/hmc/domain/model/enums/DeleteHearingStatus.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.hmc.domain.model.enums;
+
+public enum DeleteHearingStatus {
+    HEARING_REQUESTED,
+    UPDATE_REQUESTED,
+    UPDATE_SUBMITTED,
+    AWAITING_LISTING,
+    LISTED;
+
+    public static boolean chekStatus(String status) {
+        for (DeleteHearingStatus enumStatus : values()) {
+            if (enumStatus.name().equals(status)) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
```suggestion
        return Arrays.stream(values()).anyMatch(enumStatus -> enumStatus.name().equals(status));
```
Optional one-liner alternative 😄 

Same comment applies to `UpdateHearingStatus`

Reply:
changed in both places